job/runner.go

```diff
@@ -53,6 +53,18 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
+type CleanupType int
```

Comment:
__[Golint]__ exported type CleanupType should have comment or be unexported ([link](https://sider.review/gh/repos/169710725/pulls/25#issue-4259314))

<sub>posted by Sider</sub>